gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineModuleJvmArgs.java

```diff
@@ -225,8 +225,8 @@ public Iterable<String> asArguments() {
 
             // Derive this plugin's `enablePreview` property from BaselineJavaVersion's extension
             project.getPlugins().withType(BaselineJavaVersion.class, _unused -> {
-                BaselineJavaVersionsExtension javaVersionsExtension =
-                        project.getExtensions().getByType(BaselineJavaVersionsExtension.class);
+                BaselineJavaVersionExtension javaVersionsExtension =
+                        project.getExtensions().getByType(BaselineJavaVersionExtension.class);
```

Comment:
the subtle typo here is that I was detecting that the _singular_ BaselineJavaVersion plugin was applied, but trying to read the _root level_ BaselineJavaVersionsExtension off this project.

This can only work in single project builds.